app/views/curation_concerns/file_sets/_single_use_link_rows.html.erb

```diff
@@ -0,0 +1,16 @@
+<% single_use_links.each_with_index do |link, index| %>
+  <% type = link.path =~ /downloads/ ? 'Download' : 'Show' %>
```

Comment:
@mjgiarlo I think it's more akin to presenter logic. I don't have any problems with creating a presenter. That would the be _proper_ way, or we could use helper methods.